.vscode/launch.json

```diff
@@ -14,6 +14,7 @@
                 "--kubeconfig=${workspaceFolder}/.kubeconfig",
                 "--webhook-bind-address=:2443",
                 "--webhook-tls-directory=${workspaceFolder}/.local/ssl",
+                "--enableWebhooks=false",
```

Comment:
disabling webhooks will break the controller since it uses mutating webhooks...

Reply:
that's why the webhooks are enabled by default, but can be easily disabled for local development.